.github/workflows/main.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       run: |
         ./build config                   \
                 model=${{ matrix.model}} \
-                name=CRUEL-V3.4          \
+                name=VN™kERNEL (Cruel)         \
```

Comment:
You need to add quotes if you use whitespaces, like name="VN™kERNEL (Cruel)". It's better not to use white spaces at all in name.